tools/autotools/tsk_logical_imager.cpp

```diff
@@ -635,15 +627,31 @@ BOOL getDriveToProcess(string& driveToProcess) {
     }
 }
 
-int main(int argc, char **argv1) {
-    TSK_IMG_INFO *img;
-    TSK_IMG_TYPE_ENUM imgtype = TSK_IMG_TYPE_DETECT;
+static void usage() {
+    TFPRINTF(stderr,
+        _TSK_T
+        ("usage: %s [-i imgPath] -c configPath\n"),
+        progname);
+    tsk_fprintf(stderr, "\t-i imgPath: The image file\n");
+    tsk_fprintf(stderr, "\t-c configPath: The configuration file\n");
+    tsk_fprintf(stderr, "\t-v: verbose output to stderr\n");
+    tsk_fprintf(stderr, "\t-V: Print version\n");
+    exit(1);
+}
+
+int
+main(int argc, char **argv1)
+{
+	TSK_IMG_INFO *img;
```

Comment:
It looks like there are tabs instead of spaces in this file.

Reply:
sorry. Not probably cut & paste. Untabify will be use from now on :-)